influxdb3_server/src/lib.rs

```diff
@@ -206,6 +212,9 @@ where
     let hybrid_make_service = hybrid(rest_service, grpc_service);
 
     let addr = AddrIncoming::from_listener(server.listener)?;
+    let timer_end = Instant::now();
+    let startup_time = timer_end.duration_since(startup_timer);
+    info!("Server Startup Time: {}ms", startup_time.as_millis());
```

Comment:
maybe better to output this using duration's formatting? I think it'll do things like 1m32s...

Reply:
@pauldix it does not have a formatting available to it. This is why you need to specify how you want it, and mostly it seems to do fractional (e.g. 2.7 seconds) rather than absolute (e.g. 2s 700ms) for the various methods. I figured ms was the most accurate, but I'm willing to do [this method instead](https://www.chiark.greenend.org.uk/~ijackson/quicksand/d/doc/tokio/time/struct.Duration.html#method.as_secs_f64) to get something like 2.7s or 800.5s

Reply:
nah, just keep the milliseconds. Perhaps I was thinking for Duration's debug output. Or maybe a Chrono duration to string.